components/summary/summary--total.hbs

```diff
@@ -0,0 +1,49 @@
+<div class="summary">
+  <h2 class="summary__title neptune">Summary - Section Title</h2>
+  <div class="summary__block">
+    <div class="summary__item">
+      <div class="grid">
+        <div class="grid__col col-12@xs col-6@m">
+          <div class="summary__question mars">Total value of acquisitions for transport assets and equipment</div>
+        </div>
+        <div class="grid__col col-8@xs col-4@m">
+          <div class="summary__answer summary__answer--total venus">
+            &pound;9000.00
+          </div>
+        </div>
+        <div class="grid__col col-4@xs col-2@m">
+          <div class="summary__link mars">
+            <a href="#">Change <span class="u-vh">your answer for: Total value of acquisitions for transport assets and equipment. The current value is: 9000.00</span></a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="summary__item">
+      <div class="grid">
+        <div class="grid__col col-12@xs col-6@m">
+          <div class="summary__question mars">Total value of acquisitions for computers and peripheral devices (hardware)</div>
+        </div>
+        <div class="grid__col col-8@xs col-4@m">
+          <div class="summary__answer summary__answer--total venus">
+            &pound;234,000.00
+          </div>
+        </div>
+        <div class="grid__col col-4@xs col-2@m">
+          <div class="summary__link mars">
+            <a href="#">Change <span class="u-vh">your answer for: Total value of acquisitions for computers and peripheral devices (hardware). The current value is: 234,000.00</span></a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="summary__item">
```

Comment:
<img width="607" alt="summary-title-borders-2px" src="https://user-images.githubusercontent.com/22910336/43196673-ad38a18a-9000-11e8-8c20-57a5a3a6ad9e.png">

This section should have **2px borders** to differentiate it from the editable sections. 

Suggest `.summary__item .summary__item--total`  and `.summary__block .summary__block--totalling` or suchlike to handle this.